src/restconfig/src/test/java/org/geoserver/rest/LoggingControllerTest.java

```diff
@@ -44,12 +45,14 @@ public void reset() throws Exception {
     @Test
     public void testGetLoggingAsJSON() throws Exception {
         JSON json = getAsJSON(RestBaseController.ROOT_PATH + "/logging.json");
-        print(json);
+        if (LOGGER.isLoggable(Level.FINE)) {
```

Comment:
Probably best leaving it as it was before...
Before: the print method internally checks for quiet tests, won't report anything during a build, but would dump info when running from IDE (when a developer most needs it).
After: dumps nothing regardless,  unless one also sets the logging profile to GEOSERVER_DEVELOPER_LOGGING. Seems overkill..

Reply:
I would like to improve this method name to something more specific then, a method called "print" is just begging to be commented out or removed :)

Reply:
That is probably best discussed on the devel list, so that everyone at least knows about it. Those print methods have been there for something like 15 years (thinking about it sibling printing a XML DOM) with no other complaint.